dabudimesh/main.py

```diff
@@ -1,23 +1,16 @@
 from router import Router
-from utils import create_server, socket_address
-from shell import DabudiShell, _on_read
+from shell import DabudiShell
 import asyncio
-
-
-def _on_accept(router):
-    (sock, _) = router.accept()
-    asyncio.get_event_loop().add_reader(sock, _on_read, sock, router)
+import random
 
 
 def main():
-    listener = create_server()
-    address = socket_address(listener)
-    router = Router(address, listener)
+    address = str(random.randint(1500, 5000))
+    router = Router(address)
```

Comment:
How are you going to use Bluetooth MAC addresses with this?

Reply:
When we use TCP/IP for connections in our local network we use port as address and IP address is the same. When using Bluetooth we use MAC address and port remains the same.